components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.ts

```diff
@@ -74,11 +74,10 @@ export class ResourceDropdownComponent implements OnInit, OnChanges {
     if (!this.active) { // opening
       this.filterValue = '';
       this.filteredResources = this.resourcesInOrder;
+      this.active = !this.active;
```

Comment:
Because this is now used only in a single case, I think it improves clarity to do this:
```suggestion
      this.active = true;
```
(Just like there is `this.active = false;` on L95.)